crux-mir/README.md

```diff
@@ -56,7 +65,18 @@ versions corresponding to the last two `crux-mir` releases.
 [mir-json-readme]: https://github.com/GaloisInc/mir-json#readme
 
 
-## Setup: Build and Install
+## Setup: `crux-mir`
+
+### Download `crux-mir` binaries
+
+`crux-mir` comes with nightly builds and stable [binary releases](https://github.com/GaloisInc/crucible/releases). If you want to install the nightly build, do the following:
+
+* Go to the [crux-mir action](https://github.com/GaloisInc/crucible/actions/workflows/crux-mir-build.yml) page
+* Select the last build from `master` and scroll down the page, until you find and download the desired binary release (`crux-mir-macos...` or `crux-mir-ubuntu...`)
+* Unzip the downloaded archive and add it to your path
+
+
+### Build `crux-mir` from soruce
```

Comment:
Similarly, it would be worth a brief paragraph after the "Setup: `crux-mir`" section header explaining that you can either download it or build it from source.

Reply:
Done in 259b511